drivers/rtc/rtc_ll_stm32.c

```diff
@@ -323,8 +345,10 @@ static int rtc_stm32_get_calibration(const struct device *dev, int32_t *calibrat
 {
 	ARG_UNUSED(dev);
 
-	uint32_t calp_enabled = LL_RTC_CAL_IsPulseInserted(RTC);
-	uint32_t calm = LL_RTC_CAL_GetMinus(RTC);
+	uint32_t calr = sys_read32(&RTC->CALR);
+
+	bool calp_enabled = READ_BIT(calr, RTC_CALR_CALP);
+	uint32_t calm = READ_BIT(calr, RTC_CALR_CALM);
```

Comment:
Unfortunately this isn't compatible with stm32f2 series (which uses different register names).

Reply:
Fixed by [1a95dfc](https://github.com/zephyrproject-rtos/zephyr/pull/62304/commits/1a95dfc9d16324064af6ed77b895bd362c698d1b). I have disabled calibration for STM32F2 and STM32F2 Cat. 1 as I can't do any code for coarse calibration since I do only work on an H7 right now.